mart/configs/model/classifier_timm.yaml

```diff
@@ -3,10 +3,10 @@ defaults:
 
 modules:
   preprocessor:
-    # Convert [0, 255] input to [0, 1]
+    # no-op
     _target_: torchvision.transforms.Normalize
     mean: 0
-    std: 255
+    std: 1
```

Comment:
Why not just remove this?

Reply:
This is needed, as `timm` uses ImageNet statistics for data standardization.

I wasn't aware of this, because `convnext-tiny`, unlike `resent18`, is very flexible with normalization that removing standardization wouldn't affect accuracy much.

Reply:
Usually that means batchnorm is in play somehow.